aspnet-core/xml/ns-Microsoft.AspNetCore.Mvc.Razor.Compilation.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Mvc.Razor.Compilation">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types used to represent a compiled Razor Page or View.</summary>
+    <remarks>Not intended for use in application code.</remarks>
```

Comment:
Tried to determine usability in application code by contained types, please let me know if these assumptions are incorrect.